scripts/import/import_all_sv_data.pl

```diff
@@ -955,7 +963,21 @@ sub get_gender {
   return $genders{$row};
 }
 
+# Get the phenotype attrib id
+# By default, attrib = 'trait'
+sub get_attrib_id {
+  my $stmt = $dbVar->prepare(qq { SELECT attrib_id FROM attrib WHERE value = 'trait' });
+  $stmt->execute() || die;
+  my $attrib_id = ($stmt->fetchrow_array)[0];
+
+  debug(localtime()." No attribute 'trait' was found in attrib table!\n") unless defined $attrib_id;
+
+  return $attrib_id;
+}
+
 sub phenotype_feature {
+  my $class_attrib_id = shift;
+
   my $stmt;
   
   # Check if there is some phenotype entries
```

Comment:
Can the input parameter be checked that is defined?  $class_attrib_id

Reply:
get_attrib_id L968 dies if attrib_id is undef.